pkg/agent/baker.go

```diff
@@ -906,6 +906,12 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"GetOutboundCommand": func() string {
 			return getOutBoundCmd(config, config.CloudSpecConfig)
 		},
+		"BlockOutboundNetwork": func() bool {
+			if config.OutboundType == datamodel.OutboundTypeBlock || config.OutboundType == datamodel.OutboundTypeNone {
```

Comment:
are we sure we want `OutboundTypeNone` to indicate the same behavior as `OutboundTypeBlock`?

Reply:
Yes, they both mean the same to us - I've double checked this a handful of times since I think its a bit convoluted but it is what I've been told

Reply:
can we add comments to the respective definitions of these outbound types to indicate they mean the same thing?

maybe worth following up and seeing if we can eliminate one or the other altogether

Reply:
Here's a reply from Coco in one of the various NIC chats. I'll leave some commends as well but I'll drop the response in the comment for legacy sake 

What’s the difference between outboundtype=none and outboundtype=block? 
`–outbound-type=none`: AKS doesn't set up an outbound for users, but users can set the outbound rules. If there is an Azure Route Server announcing a default route via an NVA for egress to the internet. In this scenario, outboundType==none would still allow internet egress via the NVA 

`–outbound-type=block`: AKS doesn't set up an outbound for users and AKS actively adds an NSG rule to all cluster nodes to prevent egress to the internet. So users cannot set the outbound rules.  With  `–outbound-type=block` and ` ArtifactSource=Cache`, it’s the ‘zero egree cluster’. There are no outbound rules allowed on the cluster.

Reply:
the finny part is that the datamodel only contains those two attribute, so it's unclear what other value there could be